zebrad/tests/acceptance.rs

```diff
@@ -47,6 +50,13 @@ pub fn get_child_no_args() -> Result<(zebra_test::command::TestChild, impl Drop)
     ))
 }
 
+pub fn change_default_path() -> Result<()> {
+    let tmp = TempDir::new("zebrad_tests")?;
+    std::env::set_var("ZEBRAD_CACHE_DIR", tmp.path());
+
+    Ok(())
```

Comment:
Issue: This temporary directory is deleted when this function returns, and `tmp` is dropped:
https://docs.rs/tempdir/0.3.7/tempdir/struct.TempDir.html

Deleting the directory before the test finishes could cause weird test failures, particularly on Windows, which is more strict about file and directory access.

Instead, use a subdirectory of the existing temporary directory in `test_cmd`:
https://github.com/ZcashFoundation/zebra/blob/f2d7bb3177d5df59d26746d68ae354986372f3a7/zebra-test/src/command.rs#L10
This temporary directory is kept around by the `dir` and `_guard` variables, until the test finishes.

It's ok to always set `ZEBRAD_CACHE_DIR` in `test_cmd`.

Reply:
Thanks and sorry for not checking the variable was going to be dropped. Make the suggested changes at https://github.com/ZcashFoundation/zebra/pull/808/commits/a7142d9160fa8ad89d6e1d46974ae454885ecc66